sql/migrations/2023-05-15-work-types/up.sql

```diff
@@ -0,0 +1,179 @@
+		
+		(17,'Introductory text','Text that precedes the main work and offers some sort of introduction to it.',8,0),
+		(18,'Conclusion','Text placed after the main work and offering a conclusion to the book.',8,1),
+		(19,'Letter','Written message addressed to a person or organization. This work type should be used for real (not fictional) letters only. (Epistolary novels are novels and should not be split into individual letters.)',8,2),
+		(20,'Essay','Piece of writing in which the author develops their own argument on some subject.',8,3),
+		(21,'Speech','Address delivered to an audience (the written work being the text meant to read or the transcript of such as address).',8,4),
+		(22,'Scientific literature','Scholarly work containing firsthand reports of research, often reviewed by experts (primary literature), or synthesizing and condensing what is known on specific topics (secondary literature).',8,5),
+		(23,'Biographical literature','Work describing a real person’s life.',8,7),
+		(24,'Reference work','Informative work intended for consultation rather than consecutive reading.',8,8),
+		(25,'Legal instrument','A formal written legal document.',8,9),
+		(26,'Recipe','A set of instructions for making a dish of prepared food. Recipes are generally preceded by the list of necessary ingredients.',8,10),
+		
+		(27,'Sonnet','A 14-line poem with a variable rhyme scheme originating in Italy, and originally consisting of two quatrains and two tercets. Traditionally, a volta occurs between the eighth and ninth lines (or before the final couplet in the Shakespearean sonnet).',4,1),
+		(28,'Ballad','Short narrative poem in rhythmic verse suitable for singing, often in quatrains and rhyming the second and fourth lines.',4,2),
+		(29,'Haiku','Originally, a traditional short Japanese poetic form with a 5-7-5 phonetic units pattern, now adapted in different ways in other languages.',4,3),
+		(30,'Villanelle','Poetic composition consisting of nineteen lines: five tercets followed by a quatrain, with the first and third lines of the first stanza repeating alternately in the following stanzas, forming refrains, and as the final two lines of the final quatrain.',4,4);
+
+COMMIT;
+
+
+	
+-- Finally, third level types
+BEGIN TRANSACTION;
+
+	-- Short Story
+	UPDATE bookbrainz.work_type
+	SET description='Prose narrative that is shorter than a novel or novella and that usually deals with only a few characters.',
+		parent_id=14,
+		child_order=0
+	WHERE id=2;
+	-- Novella
+	UPDATE bookbrainz.work_type
+	SET description='Prose narrative whose length is shorter and less complex than most novels, but longer and more complex than most short stories.',
+		parent_id=14,
+		child_order=1
+	WHERE id=12;
+	-- Novel
+	UPDATE bookbrainz.work_type
+	SET description='Prose narrative of considerable length and a certain complexity.',
+		parent_id=13,
+		child_order=1
+	WHERE id=1;
```

Comment:
In your screenshot I can see that you originally had novel as a child of short-form fiction (14). Since you've already corrected it to be a direct child of fiction (13), it is now a second level type, no longer a third level type. So I guess this should be moved into the previous SQL transaction although it does not really matter as it has no children.